src/reformed.js

```diff
@@ -45,11 +46,26 @@ const makeWrapper = (middleware) => (WrappedComponent) => {
       }
     }
 
-    bindInput = (name) => {
+    bindInput = (name, opts) => {
+      const cfg = assign({
+        updateOn: 'onChange',
+        debounce: 0
+      }, opts)
+
+      const { updateOn } = cfg
+      const asyncHandler = (fn) => (e) => {
```

Comment:
Would be nice to find some way to cache this function (presumably like https://github.com/acdlite/recompose/blob/master/docs/API.md#withhandlers) so that we don't interfere with shallow prop comparisons. Not a blocker, but would be nice to consider if it doesn't lead to an obtuse API.

Theoretically would be difficult since it would require memoizing the `bindInput` function for multiple arguments, and would also require that the implementer always pass the same object in. So, pretty much unrealistic, but food for thought.